### ISSUE TITLE
Split the guide to US and non-US content

### ISSUE THREAD

**Issue by bz123** [jlevy/og-equity-compensation#26]
Great guide, would be awesome if you could split the guide where international content is available, so that people outside of the US could use that as a future reference without having to go through specifics in the US law - taxation etc.

**Comment by kspaans** [comment]
I'm interested in helping with this (I'm a Canadian working in the UK, so I've got two jurisdictions I want to know about!). Would a first step perhaps be to add subsections that mention equivalent (if they exist) vehicles from other countries?

**Comment by mwarkentin** [comment]
+1 for Canada :maple_leaf:

**Comment by anilmujagic** [comment]
+1 for Germany

**Comment by jlevy** [comment]
Agreed, it would be great if we can do this. It's also a bit complex and requires a lot of expertise.

The best approach would be to get a knowledgeable contributor for each country to start writing a new file with everything that's different and specific to that country. Once we have one or two examples (say Canada and Germany), I think it'll become clearer how to reorg the US items. But in the meantime, until we have content ready for new countries, let's keep the main file US-focused.

Added #35 for Canada. If someone has the skills and motivation to do that one, drop a comment there! Feel free to create additional issues for additional countries if desired.

**Comment by gsakkis** [comment]
+1 for UK
